system_baseline/views/v1.py

```diff
@@ -261,6 +265,29 @@ def get_baselines(limit, offset, order_by, order_how, display_name=None):
     )
 
 
+def check_dirty_baselines(baselines):
+    auth_key = get_key_from_headers(request.headers)
+    for baseline in baselines:
+        for system_id in baseline.mapped_system_ids():
+            try:
+                # fetch system from inventory
+                message = "read system with profiles"
+                current_app.logger.audit(message, request=request)
+                fetch_systems_with_profiles(
+                    [system_id], auth_key, current_app.logger, get_event_counters()
+                )
+            except ItemNotReturned:
+                # not in inventory => delete in our db
+                try:
+                    baseline.remove_mapped_system(system_id)
```

Comment:
Thanks for this PR!

When testing, I could not have the association deletion working.

I needed to add the following to get it working :) 

```suggestion
                    baseline.remove_mapped_system(system_id)
                    account_number = view_helpers.get_account_number(request)
                    SystemBaselineMappedSystem.delete_by_system_ids([system_id], account_number)
```

Reply:
You are right... `remove_mapped_system` do not commit changes. My bad. I will use `delete_by_system_ids` as this commits changes and wants `account_number`.